src/hb-ot-color-sbix-table.hh

```diff
@@ -198,7 +198,7 @@ struct sbix
 
     struct PNGHeader
     {
-      HBUINT8 header[9];
+      HBUINT8 header[16];
```

Comment:
Where does 16 comes from!?  8.  8 is the number!

Reply:
https://github.com/behdad/color-emoji/blob/master/png.py#L81

Reply:
I'm sorry.

Reply:
There's 8 byte PNG header, then 8 byte IHDR chunk header:
https://github.com/behdad/color-emoji/blob/master/png.py#L69

Having 16 is fine I guess. Sorry about that.  Thanks.  Or make it signature[8] and chunkHeaderIHDR[8].

Reply:
For my OCD, maybe add all the IHDR fields:
```		#  Width:              4 bytes
		#  Height:             4 bytes
		#  Bit depth:          1 byte
		#  Color type:         1 byte
		#  Compression method: 1 byte
		#  Filter method:      1 byte
		#  Interlace method:   1 byte
```

Reply:
Looks nice! Done. Thanks